### PR TITLE
zov add buchungsnummer weitere person

### PR DESCRIPTION
**Pull request by judithmh**
- added the input field buchungsnummer to weitere person 
- added the option for helper text to input fields

### CODE REVIEW

File: app/flows/fluggastrechteFormular/persoenlicheDaten/__test__/doneFunctions.test.ts [hunk 1]
```diff
@@ -55,6 +55,26 @@ describe("navStates", () => {
 
       expect(actual).toBe(true);
     });
+
+    it("should return true given isWeiterePersonen yes and weiterePersonen with buchungsnummer", () => {
```
Comment (pgurusinga) [inline app/flows/fluggastrechteFormular/persoenlicheDaten/__test__/doneFunctions.test.ts:59]: can we add another case where `should return true given isWeiterePersonen yes and weiterePersonen "without" buchungsnummer`

Comment (judithmh) [reply app/flows/fluggastrechteFormular/persoenlicheDaten/__test__/doneFunctions.test.ts:59]: It is already exisiting above this test :)

File: app/components/inputs/Input.tsx [hunk 2]
```diff
@@ -16,9 +16,9 @@ export type InputProps = Readonly<{
   prefix?: string;
   suffix?: string;
   errorMessages?: ErrorMessageProps[];
-  helperText?: string;
   width?: FieldWidth;
   formId?: string;
+  helperText?: string;
```
[the remark this answers is not in the document]
Comment (judithmh) [reply app/components/inputs/Input.tsx:21]: I added one :)

**Review by aaschlote — APPROVED** [review]
LGTM

**Pull request merged**